python/paddle/fluid/contrib/slim/quantization/quantization_mkldnn_pass.py

```diff
@@ -557,13 +560,16 @@ def _transform_to_quantize_mkldnn(self, graph, op_node):
         graph.safe_remove_nodes(op_node)
         return quant_op_node
 
-    def _transform_to_mul_mkldnn(self, graph, op_node, quantize_node):
-        input_name = op_node.input("X")[0]
-        scale_in = quantize_node.op().attr("Scale")
-        op_node.set_attr("scale_y", [1.0])
-        op_node.set_attr("scale_x", scale_in)
-        op_node.set_attr("scale_out", 1.0)
-        op_node.set_attr("force_fp32_output", True)
+    def _update_conv_relu_scales(self, graph):
+        for op in graph.all_op_nodes():
+            if op.name() in self._conv_ops:
+                out_name = op.output("Output")[0]
+                if out_name in self._var_quant_scales and \
+                   op.op().attr("fuse_activation") == 'relu' and \
+                   op.op().attr("fuse_residual_connection") == False:
+                    _, tensor = self._var_quant_scales[out_name]
+                    self._var_quant_scales[out_name] = (True, tensor)
```

Comment:
Here, you just set the first flag as `True`, but don't change the scale value. Is it right?

Reply:
Yes, here I pass the normalized scale. The actual rescaling happens here:
https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc#L46-L47